backend/main.py

```
@@ -0,0 +1,31 @@
+from fastapi import FastAPI, HTTPException
+import uvicorn
+from config import is_dev, first_setup
+import database
```

Ruff (0.11.9)

`database` imported but unused

Remove unused import: `database`

(F401)